MediaGallerySynchronizationApi/Model/FetchBatchesInterface.php

```diff
@@ -19,7 +19,7 @@ interface FetchBatchesInterface
      *
      * @param string $tableName
      * @param array $columns
-     * @param string|null $modificationDateColumn
+     * @param string $modificationDateColumn
```

Comment:
The modification date column can be null, this column needed to fetch only those entities wich was changed since last execution, so the Generator compare modification date column and if the value bigger then in last execution time they will include this item to the batch. this behavior was implemented to avoid fetch again items thats were fetched in previous synchronization

Reply:
and if you don't like fetch items with modificationDate column you can skip this behavior by putting null to this parametr

Reply:
Hi @Nazar65, thanks for the review. I'll update my pull request with the requested changes. Thank you!